watchdog/map.jinja

```diff
@@ -0,0 +1,14 @@
+{% set server = salt['grains.filter_by']({
+    'HP': {
+      'module': 'hpwdt',
+      'parameter': {},
```

Comment:
It's still the same, should be:
```
'kernel': {
  'parameter': {}
}
```

Reply:
Sorry - misunderstanding - should be fixed now.